build.macos64x64/third-party/libgit2/Makefile.lib

```diff
@@ -0,0 +1,43 @@
+include ../common/Makefile.lib.extra
+
+$(info LIBRARY: libgit2)
+
+# plugin definitions
+LIBGIT2URL:=https://github.com/libgit2/libgit2/archive/v0.23.0.tar.gz
+LIBGIT2DIR:=$(THIRDPARTYDIR)/libgit2-0.23.0
+LIBGIT2LIBNAME:=libgit2.0.23.0.dylib
+LIBGIT2LIB:=$(THIRDPARTYINSTALLDIR)/$(LIBGIT2LIBNAME)
+
+# ensure third-party library is built and recognised by plugins
+INCDIRS:=$(INCDIRS) $(THIRDPARTYINCLUDEDIR)
+EXTRALIBS:=$(EXTRALIBS) $(LIBGIT2LIB)
+PLUGINREQS:=$(THIRDPARTYLIBS)
+
+$(LIBGIT2DIR)/CMakeLists.txt:
+	curl -L $(LIBGIT2URL) | tar x -C $(THIRDPARTYDIR)
+	touch $(LIBGIT2DIR)/CMakeLists.txt
+
+$(LIBGIT2LIB): $(LIBGIT2DIR)/CMakeLists.txt
+	cd $(LIBGIT2DIR) \
+		&& cmake \
+			-DCMAKE_INSTALL_PREFIX=$(THIRDPARTYOUTDIR) \
+			-DUSE_SSH=ON \
+			-DBUILD_CLAR=OFF \
+			-DLIBSSH2_FOUND=ON \
+			-DLIBSSH2_INCLUDE_DIRS=$(THIRDPARTYINCLUDEDIR) \
+			-DLIBSSH2_LIBRARIES=$(THIRDPARTYLIBDIR)/libssh2.1.dylib \
+			-DCMAKE_C_FLAGS=-m64 \
+			-DCMAKE_CXX_FLAGS=-m64 \
+			-DCMAKE_OSX_ARCHITECTURES=x86_64 \
+			-DCMAKE_SHARED_LINKER_FLAGS=-Wl,-rpath,@executable_path:@executable_path/Plugins \
```

Comment:
afaiu this, that line is telling cmake to use rpath... so what's the observation here?